### ISSUE TITLE
Health bars leaving static duplicates behind.

### ISSUE THREAD

**Issue by 0beron** [brandonmousseau/vhvr-mod#107]
Sometimes healthbars can leave copies behind. These float around without their associated player or mob. They no longer billboard, and you can view them from the reverse side. They also seem to draw on top of everything, even over large distances through trees/hills.

Observed this with a Greyling healthbar, and some multiplayer character names/healthbars. Not certain what triggers it, could be after resting/sleep, or after connectivity issues / server disconnection.

**Comment by brandonmousseau** [comment]
Yes this is a known issue but it isn't totally clear what the root cause is yet and I have never been able to reproduce it - were you using portals when it happened by chance? Were you going to any specific biome?

**Comment by 0beron** [comment]
No portals, none have been built at all on the world I saw it on.
I don’t remember it being while crossing biomes. The greyling one could have been near black forest/meadows border, but the player names got duplicated inside a house we built within sight of the stone circle at the start.

**Comment by 0beron** [comment]
Saw this twice again the other day, and both times it happened immediately after becoming disconnected from a multiplayer game. The duplicate names/ health bars were frozen wherever the other players were on disconnect, and would persist after reconnection, although I did not restart my game executable at any point.
